genon/preprocessor/facade/convert_processor.py

```
@@ -813,6 +822,30 @@ def convert_to_pdf(file_path: str, use_pdf_sdk: bool = True) -> str | None:
+    """
+    try:
+        from genon.preprocessor.converters.hwp_to_pdf.availability import (
+            libreoffice_available,
+            pdf_sdk_available,
+            rhwp_available,
+        )
+        return bool(pdf_sdk_available() or rhwp_available() or libreoffice_available())
+    except ImportError:
+        # facade 단일 파일 실행 등으로 모듈 import 가 안 되는 경우 → 기존 동작 유지(가용 가정)
+        return True
+    except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)